dbms/src/Storages/Page/V3/Universal/UniversalPageIdFormat.h

```diff
@@ -31,8 +31,9 @@ namespace DB
 //
 //
 // The main key format types:
-// Raft related key
+// Raft related key format
 //  Format: https://github.com/tikv/tikv/blob/9c0df6d68c72d30021b36d24275fdceca9864235/components/keys/src/lib.rs#L24
+//  And to distinguish data written by kv engine and raft engine, we prepend an `0x01` to the key written by kv engine.
```

Comment:
I didn't get it. Does not all keys written by raft engine has `LOCAL_PREFIX`(0x01)? How can you distinguish the key written by raft engine and kv engine. 
https://github.com/tikv/tikv/blob/9c0df6d68c72d30021b36d24275fdceca9864235/components/keys/src/lib.rs#L24

Reply:
And some more comments about it.